compass/ocean/tests/mitgcm_baroclinic_gyre/gyre_test_case.py

```diff
@@ -0,0 +1,61 @@
+from compass.mesh import QuasiUniformSphericalMeshStep
+from compass.ocean.tests.mitgcm_baroclinic_gyre.cull_mesh import CullMesh
+from compass.ocean.tests.mitgcm_baroclinic_gyre.forward import Forward
+from compass.ocean.tests.mitgcm_baroclinic_gyre.initial_state import (
+    InitialState,
+)
+from compass.testcase import TestCase
+
+
+class GyreTestCase(TestCase):
+    """
+    A class to define the MITgcm baroclinic gyre test cases
+
+    Attributes
+    ----------
+    resolution : str
+        The resolution of the test case
+    """
+
+    def __init__(self, test_group, resolution, long):
+        """
+        Create the test case
+
+        Parameters
+        ----------
+        test_group :
+        compass.ocean.tests.mitgcm_baroclinic_gyre.MitgcmBaroclinicGyre
+            The test group that this test case belongs to
+
+        resolution : str
+            The resolution of the test case
+
+        long : bool
+            Whether to run a long (3-year) simulation to quasi-equilibrium
+        """
+        name = 'performance'
+        self.resolution = resolution
+        self.long = long
+
+        if long:
+            name = 'long'
```

Comment:
We've sometimes called these something like `performance_test` and `3_year_test`. You might consider changing it after deciding how long the test should be. I know the soma test case uses `long` but I think it's nice to know as a user how long it will be.

Reply:
Added @cbegeman's suggestion.